### ISSUE TITLE
[BUG] new volume create fails with "Unauthorized"

### ISSUE THREAD

**Issue by stenwt** [longhorn/longhorn#3650]
[longhorn-support-bundle_8b12c0fa-0279-4190-9995-052abb72b3bd_2022-02-22T18-31-27Z.zip](https://github.com/longhorn/longhorn/files/8121631/longhorn-support-bundle_8b12c0fa-0279-4190-9995-052abb72b3bd_2022-02-22T18-31-27Z.zip)
## Describe the bug

I recently added a new node to my k3s cluster. The node has healthy longhorn replicas on it, but when a new volume was to be created on that node, it failed with: 

```
80s         Warning   ProvisioningFailed       persistentvolumeclaim/teleport-cluster           failed to provision volume with StorageClass "longhorn": rpc error: code = Internal desc = Bad response statusCode [500]. Status [500 Internal Server Error]. Body: [code=Server Error, detail=, message=unable to create volume: unable to create volume pvc-953bd9d9-5b39-45b2-9d03-2e9578626319: Unauthorized] from [http://longhorn-backend:9500/v1/volumes]
```

## To Reproduce

3. Perform '....'

## Log or Support bundle

If applicable, add the Longhorn managers' log or support bundle when the issue happens. 
You can generate a Support Bundle using the link at the footer of the Longhorn UI.

## Environment

 - Longhorn version: 1.2.3
 - Installation method (e.g. Rancher Catalog App/Helm/Kubectl): kubectl
 - Kubernetes distro (e.g. RKE/K3s/EKS/OpenShift) and version: k3s  v1.22.6+k3s1
   - Number of management node in the cluster: 4 (untainted)
   - Number of worker node in the cluster: 0
 - Node config
   - OS type and version: Fedora IoT  35.20220218.0 
   - CPU per node: 4
   - Memory per node: 16GB
   - Disk type(e.g. SSD/NVMe): NVMe
   - Network bandwidth between the nodes: 1Gbps
 - Underlying Infrastructure (e.g. on AWS/GCE, EKS/GKE, VMWare/KVM, Baremetal): baremetal
 - Number of Longhorn volumes in the cluster: 4

**Comment by stenwt** [comment]
I drained and cordoned the node, and was able to create a pvc. where can I upload a support bundle?

**Comment by derekbit** [comment]
You can upload the support bundle here or via UI to send us.

**Comment by stenwt** [comment]
this looks to have been caused by the new node not having NTP enabled.

**Comment by derekbit** [comment]
Some warnings and errors found in your log
- No available disk for replica
``` 
2022-02-23T02:30:58.450925561Z time="2022-02-23T02:30:58Z" level=error msg="There's no available disk for replica pvc-5ccf7d90-a66e-4740-835e-777b3a889a22-r-3e0cb9f6, size 5368709120"
2022-02-23T02:30:58.450936924Z time="2022-02-23T02:30:58Z" level=error msg="unable to schedule replica of volume with HardNodeAffinity = hpt1.sud0.com" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=hpt1.sud0.com owner=hpt1.sud0.com replica=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22-r-3e0cb9f6 state=attached volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22
2022-02-23T02:30:58.559505947Z time="2022-02-23T02:30:58Z" level=debug msg="Found 1 use zones []" accessMode=rwo controller=longhorn-volume frontend=blockdev migratable=false node=hpt1.sud0.com owner=hpt1.sud0.com replicaAutoBalanceType=zone state=attached volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22
```

- Time skewed
It should be fixed by enabling the NTP as you mentioned.
```
2022-02-22T12:30:51.604262272-06:00 time="2022-02-22T18:30:51Z" level=error msg="Error inspecting backup config" backup=backup-d75c71cb5dc64761 controller=longhorn-backup error="error getting backup config s3://longhorn@us-east-1/?backup=backup-d75c71cb5dc64761&volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22: Failed to execute: /var/lib/longhorn/engine-binaries/longhornio-longhorn-engine-v1.2.3/longhorn [backup inspect s3://longhorn@us-east-1/?backup=backup-d75c71cb5dc64761&volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22], output failed to list objects with param: {\n  Bucket: \"longhorn\",\n  Delimiter: \"/\",\n  Prefix: \"/\"\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\n403 \n\n, stderr, time=\"2022-02-22T18:30:51Z\" level=error msg=\"Fail to list s3: failed to list objects with param: {\\n  Bucket: \\\"longhorn\\\",\\n  Delimiter: \\\"/\\\",\\n  Prefix: \\\"/\\\"\\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\\n403 \\n\" pkg=s3\ntime=\"2022-02-22T18:30:51Z\" level=error msg=\"failed to list objects with param: {\\n  Bucket: \\\"longhorn\\\",\\n  Delimiter: \\\"/\\\",\\n  Prefix: \\\"/\\\"\\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\\n403 \\n\"\n, error exit status 1" node=hpt3.sud0.com
2022-02-22T12:31:21.522116800-06:00 time="2022-02-22T18:31:21Z" level=error msg="Error inspecting backup config" backup=backup-d75c71cb5dc64761 controller=longhorn-backup error="error getting backup config s3://longhorn@us-east-1/?backup=backup-d75c71cb5dc64761&volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22: Failed to execute: /var/lib/longhorn/engine-binaries/longhornio-longhorn-engine-v1.2.3/longhorn [backup inspect s3://longhorn@us-east-1/?backup=backup-d75c71cb5dc64761&volume=pvc-5ccf7d90-a66e-4740-835e-777b3a889a22], output failed to list objects with param: {\n  Bucket: \"longhorn\",\n  Delimiter: \"/\",\n  Prefix: \"/\"\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\n403 \n\n, stderr, time=\"2022-02-22T18:31:21Z\" level=error msg=\"Fail to list s3: failed to list objects with param: {\\n  Bucket: \\\"longhorn\\\",\\n  Delimiter: \\\"/\\\",\\n  Prefix: \\\"/\\\"\\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\\n403 \\n\" pkg=s3\ntime=\"2022-02-22T18:31:21Z\" level=error msg=\"failed to list objects with param: {\\n  Bucket: \\\"longhorn\\\",\\n  Delimiter: \\\"/\\\",\\n  Prefix: \\\"/\\\"\\n} error: AWS Error:  RequestTimeTooSkewed The difference between the request time and the server's time is too large. <nil>\\n403 \\n\"\n, error exit status 1" node=hpt3.sud0.com
```

- Unauthorized
```
2022-02-23T02:23:51.933141189Z time="2022-02-23T02:23:51Z" level=warning msg="Error syncing Longhorn node longhorn-system/nuc1.sud0.com: fail to sync node for longhorn-system/nuc1.sud0.com: Unauthorized"

2:30:52.006752442Z time="2022-02-23T02:30:52Z" level=debug msg="Failed to check for the latest and stable Longhorn versions" controller=longhorn-setting error="failed to get Kubernetes server version: the server has asked for the client to provide credentials" node=nuc1.sud0.com
```
